### PR TITLE
#2347: First hack of ignoring already tagged items

### PR DESCRIPTION
**Pull request by SusannaMaria**
Improve the speed of fetching acousticbrainz data. Ignore already existing data.

### CODE REVIEW

File: beetsplug/acousticbrainz.py [hunk 1]
```diff
@@ -151,10 +158,17 @@ def _get_data(self, mbid):
...
 
-    def _fetch_info(self, items, write):
+    def _fetch_info(self, items, write, force):
         """Fetch additional information from AcousticBrainz for the `item`s.
         """
         for item in items:
+            
+            if not force:
+                mood_str = item.get('mood_acoustic', u'')
+                if len(mood_str) != 0:
```
Comment (Kraymer) [inline beetsplug/acousticbrainz.py]: or simply 
``if mood_str:``

File: beetsplug/acousticbrainz.py [hunk 2]
```diff
@@ -118,7 +119,13 @@ def commands(self):
 
         def func(lib, opts, args):
             items = lib.items(ui.decargs(args))
-            self._fetch_info(items, ui.should_write())
+            self._fetch_info(items, ui.should_write(),opts.force_refetch or self.config['force'])
```
Comment (Kraymer) [inline beetsplug/acousticbrainz.py]: ,opts.force_refetch => , opts.force_refetch

File: beetsplug/acousticbrainz.py [hunk 3]
```diff
@@ -118,7 +119,13 @@ def commands(self):
```
Comment (sampsyo) [inline beetsplug/acousticbrainz.py]: I think we might need something like this to make the command-line flag work:

```py
cmd.parser.add_option(
    u'-f', u'--force', dest='force_refetch',
    action='store_true', default=False,
    help=u're-download data if present',
)
```

**Comment by sampsyo** [comment]
Looks perfect! Thank you! :sparkles: :rocket: :robot:

I've merged this for inclusion in the next release.

**Pull request merged**